### DISCUSSION TITLE
cloudflare issue

### DISCUSSION THREAD

**Discussion by kwwwwang** [cefsharp/CefSharp#4556] [Q&A]
i can't pass cloudflare since yesterday, are there any changes? 
how can i pass cloudflare?

**Comment by zmcx16** [comment]
me too, and I tried use chromium 114.0.5734.0, it is work fine
https://vikyd.github.io/download-chromium-history-version/#/
![image](https://github.com/cefsharp/CefSharp/assets/7957361/d20bd148-19a3-4569-a7a7-8f50fe69a60d)

but I used CefSharp.Wpf.Example, it verify loop infinitly:
![image](https://github.com/cefsharp/CefSharp/assets/7957361/4a730ec9-98d0-4fe1-8aab-816dfa6f9add)

test website:
https://proxyrarbg.org/torrents.php

**Comment by nabkawe** [comment]
Same problem on my end, It's brand new...

**Comment by amaitland** [comment]
This isn't a `CefSharp` specific issue, I'd suggest joining in the `upstream` conversation at https://github.com/chromiumembedded/cef/issues/3547

**Comment by mwpowellhtx** [comment]
Chiming in here... the application I was previously puppeteering with some success, just recently moved hosts. With it has apparently put up a CF 'human' challenge. This is new, but it is also interfering with the sign in story. Wondering what others might have done to work through or around this?

**Reply by mwpowellhtx** [reply]
The main issue is, cannot create a context, because of some multiple ids or something getting in the way. So we could not even think about perhaps puppeteering.

```
CefSharp.Dom.MessageException: 'Protocol error (Page.enable): Duplicate `id` in protocol request'
```

**Reply by mwpowellhtx** [reply]
@amaitland Any thoughts how we could workaround?
